adb_android/adb_android.py

```diff
@@ -34,7 +34,7 @@ def bugreport(dest_file="default.log"):
     try:
         dest_file_handler = open(dest_file, "w")
     except IOError:
-        print "IOError: Failed to create a log file"
+        print("IOError: Failed to create a log file")
```

Comment:
Is this change compatible with Python 2.7.15?

Reply:
Yes, I'm pretty sure. Parenthesis are optional in print statements for Python 2.7, but required for Python 3+.

Source: https://stackoverflow.com/questions/6182964/why-is-parenthesis-in-print-voluntary-in-python-2-7

Also, I would recommend supporting Python 3+ anyways moving forward because Python 2.7 is old.

Reply:
You could also add `from __future__ import print_function` at the top to guarantee compatibility (optional).

Reply:
@anselmoshim, thanks!

Could you then please add "from __future__ import print_function" to be on a safe side?

Thanks!

Reply:
@vmalyi done, committed below.